### ISSUE TITLE
[Boss] Prince Keleseth 

### ISSUE THREAD

**Issue by Necko23** [WoWManiaUK/Redemption#2347]
While trying to do achievement "On The Rocks" - Defeat Prince Keleseth in Utgarde Keep on Heroic difficulty without shattering any Frost Tombs, I've noticed that the Tombs break by themselves. I've tried it few times, with different groups and different people. At first we thought someone was shattering them on accident, but we were wrong. I think it's bugged, we were unable to figure out the cause of them shattering. We were putting pets on Passive, dismissing them, still no results.

**Comment by Dranah** [comment]
"Break by themselves" does that mean they die or that they simply disappear?
If they are shattered, as in killed, there must be a source of damage that does so. It can be seen on combat log/recount. They are rather vulnerable to passive effects such as Wandering Plague, if you happened to run with an unholy Dk.
On the other hand, Frost tombs simply despawn after given period of time. So they just "disappear", but this does not affect the achievement status.

**Comment by fvicaria** [comment]
Can you check it in dev please?

**Comment by Necko23** [comment]
We could not find the source of damage. Even Recount didn't solve it. We were also testing if pets were shattering it by mistake, but as I said, we got pets on Passive later on, even dismissing them but yet they were disappearing.
Frost Tombs: 
Casts Frost Tomb. Immobilizes the target for 20 seconds and causes 400 damage per second (increasing to approximately 2000 damage per second in heroic mode). Frost Tombs have roughly 2,500 HP and can be DPSed. Mages can Blink out of the prison, and it is immune to Frost damage.
It DOES NOT last 20 seconds. They disappear at 5 to 10 seconds I would say. It even happened on me. I got trapped in Frost Tomb and it immediately disappeared without even anyone having the time to DPS it even if they wanted to
